### PR TITLE
Space Starter v2.0

### PR DESCRIPTION
**Pull request by AlveLarsson**
image

This is an overhaul to the space template, by @themattempire and I, improving visual quality, slight folder restructures to make it easier to understand and use.

New! Player spawn point vis, helps builders debug where the players spawn and if it has been incorrectly placed.
Added compression to player model (67 mb save)

### CODE REVIEW

File: Assets/Settings/SpaceLightingSettings.lighting [hunk 1]
```diff
@@ -27,7 +27,7 @@ LightingSettings:
   m_CompAOExponentDirect: 0.3
   m_ExtractAO: 0
   m_MixedBakeMode: 0
-  m_LightmapsBakeMode: 1
+  m_LightmapsBakeMode: 0
```
Comment (theAlexPatin) [inline Assets/Settings/SpaceLightingSettings.lighting:30]: Should this be turned off?

File: README.md [hunk 2]
```diff
@@ -1,10 +1,4 @@
...
-  <a href="https://mona.gallery" title="MoNA">
-    <img src="https://res.cloudinary.com/mona-gallery/image/upload/v1630525523/branding/logo.png" width="200px" alt="MoNA Logo"/>
-  </a>
-</p>
-
-<h1 align="center">🎨 Official MoNA Space Starter 🎨</h1>
+![splash-min](https://user-images.githubusercontent.com/61861940/147894582-30f0b076-6647-46b2-b731-1a9535b19aa9.jpg)
 <p align="center">Start building your own custom MoNA Space using our official template</p>
 
 ## ⚠️ Unity Version
```
Comment (theAlexPatin) [inline README.md:4]: I think we upgraded our Unity Version, right?

**Review by theAlexPatin — COMMENTED** [review]
Hey Alve, just a few small comments!

File: README.md (hunk 2 above)
Comment (AlveLarsson) [reply README.md:4]: No we left it at 2020.3.18 thanks to a freeze issue, it's something we can revisit unless higher priority

File: Assets/Settings/SpaceLightingSettings.lighting (hunk 1 above)
Comment (AlveLarsson) [reply Assets/Settings/SpaceLightingSettings.lighting]: Going to revert that!

**Review by theAlexPatin — APPROVED** [review]
Looks good!

**Pull request merged**